server_manager/web_app/ui_components/outline-contact-us-dialog.ts

```diff
@@ -0,0 +1,379 @@
+import {Ref, createRef, ref} from 'lit/directives/ref.js';
+import {unsafeHTML} from 'lit/directives/unsafe-html.js';
+import '@material/mwc-circular-progress';
+import '@material/mwc-radio';
+import '@material/mwc-select';
+import '@material/mwc-formfield';
+import '@polymer/paper-dialog/paper-dialog';
+import {Radio} from '@material/mwc-radio';
+import {SingleSelectedEvent} from '@material/mwc-list/mwc-list';
+import { OutlineFeedbackDialog } from './outline-feedback-dialog';
+import { PaperDialogElement } from '@polymer/paper-dialog/paper-dialog';
+import '@polymer/paper-button/paper-button';
+import * as Sentry from '@sentry/electron/renderer';
+
+import './outline-support-form';
+import {FormValues, OutlineSupportForm, ValidFormValues} from './outline-support-form';
+//import {OutlineErrorReporter} from '../../shared/error_reporter';
+import {Localizer} from '@outline/infrastructure/i18n';
+import { COMMON_STYLES } from './cloud-install-styles';
+
+/** The possible steps in the stepper. Only one step is shown at a time. */
+enum Step {
+  ISSUE_WIZARD, // Step to ask for their specific issue.
+  FORM, // The contact form.
+  EXIT, // Final message to show, if any.
+}
+
+/** Supported issue types in the feedback flow. */
+enum IssueType {
+  CANNOT_ADD_SERVER = 'cannot-add-server',
+  CONNECTION = 'connection',
+  MANAGING = 'managing',
+  GENERAL = 'general',
+}
+
+/** A map of unsupported issue types to helppage URLs to redirect users to. */
+const UNSUPPORTED_ISSUE_TYPE_HELPPAGES = new Map([
+  [IssueType.CANNOT_ADD_SERVER, 'https://support.getoutline.org/s/article/What-if-my-access-key-doesn-t-work'],
+  [IssueType.CONNECTION, 'https://support.getoutline.org/s/article/Why-can-t-I-connect-to-the-Outline-service'],
+]);
+
+@customElement('outline-contact-us-dialog')
+export class OutlineContactUsDialog extends LitElement implements OutlineFeedbackDialog {
+  static get styles() {
+    return [
+      COMMON_STYLES,
+      css`
+        :host {
+          --mdc-theme-primary: var(--primary-green);
+        }
+
+        paper-dialog {
+          width: 80%;
+        }
+
+        main {
+          display: block;
+          margin-left: auto;
+          margin-right: auto;
+        }
+
+        mwc-circular-progress {
+          left: 50%;
+          position: absolute;
+          top: 50%;
+          transform: translate(-50%, -50%);
+        }
+
+        p {
+          margin-top: .25rem;
+        }
+
+        ol {
+          list-style-type: none;
+          margin: 1.5rem 0;
+          padding-inline-start: 0;
+        }
+
+        mwc-select {
+          /**
+           * The '<app-header-layout>' restricts the stacking context, which means
+           * the select dropdown will get stacked underneath the header.
+           * See https://github.com/PolymerElements/app-layout/issues/279. Setting
+           * a maximum height will make the dropdown small enough to not run into
```

Comment:
Ugh actually I need to remove this. 99% of this code is from the client, but this particular issue actually doesn't exist on the manager since we're in a dialog.